tap_bing_ads/__init__.py

```diff
@@ -864,8 +855,7 @@ def build_report_request(client, account_id, report_stream, report_name,
     report_columns = client.factory.create(
         'ArrayOf{}Column'.format(report_name)
     )
-    getattr(report_columns, '{}Column'.format(report_name)) \
-        .append(selected_fields)
+    getattr(report_columns, '{}Column'.format(report_name)).append(selected_fields)
```

Comment:
revert this change if it is not throwing any pylint error.

Reply:
done